File: BackEnd/src/main/java/com/unosquare/admin_core/back_end/dto/ContractDto.java
package com.unosquare.admin_core.back_end.dto;

import lombok.Data;

import java.time.LocalDate;

@Data
public class ContractDTO {

    private int contractId;

    private int teamId;

    private int employeeId;

    private LocalDate startDate;

    private LocalDate endDate;
}

File: v2/AdminCore.DTOs/ContractDto.cs
using AdminCore.DTOs.Team;
using System;

namespace AdminCore.DTOs
{
  public class ContractDto
  {
    public int ContractId { get; set; }

    public int TeamId { get; set; }

    public TeamDto Team { get; set; }

    public string ClientName { get; set; }

    public int EmployeeId { get; set; }

    public DateTime StartDate { get; set; }

    public DateTime EndDate { get; set; }
  }
}